packages/jtex/src/typst/imports.ts

```diff
@@ -3,18 +3,18 @@ import type { TypstTemplateImports } from '../types.js';
 import { writeFileToFolder } from 'myst-cli-utils';
 
 export function renderTypstImports(
-  output: string,
+  output: string | false,
```

Comment:
Ah, this stops writing `myst.typ` multiple times for multi-page export. Shouldn't be `false` in other cases.